src/backend/results/get_titles/get_results.php

```
@@ -35,7 +34,7 @@ function getinprocess($missing, $code)
     return $titles;
 }
 
-function get_results($cat, $camp, $depth, $code, $filter_sparql): array
+function get_results($cat, $camp, $depth, $code, $filter_sparql, $cat2): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$camp'. (undefined)

(UnusedFormalParameter)